apps/server/lib/http/facades/query.ts

```diff
@@ -57,12 +54,7 @@ export class QueryFacade {
 				schema,
 			});
 
-			const data = await this.jellyfish.query(
-				context,
```

Comment:
To avoid confusing anyone in the future, would be nice to rename everything of type `LogContext` to `logContext` or similar. `context` is too general for such a restricted value and JF has multiple `*Context` types. Not saying it has to be like that tho